terraform/api_gateway.tf.json.template.py

```diff
@@ -241,7 +290,9 @@ def for_domain(cls, domain):
                             )
                         ]
                     }),
-                    **aws.permissions_boundary_tf
+                    # Temporally disabled until changes in this branch have been
+                    # deployed to GitLab to extend the boundary permissions.
+                    # **aws.permissions_boundary_tf
```

Comment:
We don't use comments for disabling code.

```suggestion
                    # Temporally disabled until changes in this branch have been
                    # deployed to GitLab to extend the boundary permissions.
                    **(aws.permissions_boundary_tf if False else {})
```

Reply:
Changes in the `fixup` do not match the suggestion.